[2013-05-15 05:25:41] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 05:40:35] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 05:59:03] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:13:46] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'public')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'public')
#15 [internal function]: BlogController->getView('public')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('public')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:13:51] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:30:48] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:30:52] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:34:28] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:34:33] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'blog')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'blog')
#15 [internal function]: BlogController->getView('blog')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blog')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:34:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:34:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:34:53] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:35:02] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'public')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'public')
#15 [internal function]: BlogController->getView('public')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('public')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:35:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:35:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:35:28] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'blog')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'blog')
#15 [internal function]: BlogController->getView('blog')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blog')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:35:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:35:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:42:19] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:42:19] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:42:24] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'public')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'public')
#15 [internal function]: BlogController->getView('public')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('public')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:42:28] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:47:47] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:50:07] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:50:11] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'l4')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'l4')
#15 [internal function]: BlogController->getView('l4')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('l4')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:50:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:50:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:50:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:50:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:50:41] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:51:06] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:51:41] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:51:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:51:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:52:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:52:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:52:58] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:53:04] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'public')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'public')
#15 [internal function]: BlogController->getView('public')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('public')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:53:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:53:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:54:22] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:54:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:54:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:54:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:54:51] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dd()' in /home/gristech/l4/app/routes.php:2
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 06:55:38] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dd()' in /home/gristech/l4/app/routes.php:2
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 06:57:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:51] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'public')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'public')
#15 [internal function]: BlogController->getView('public')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('public')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 06:57:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:57:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 06:58:19] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#11 [internal function]: PhotosController->index()
#12 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#13 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#14 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#16 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#17 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#18 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#19 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#20 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#21 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#22 {main} [] []
[2013-05-15 06:58:23] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dd()' in /home/gristech/l4/app/routes.php:15
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 20:00:10] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#11 [internal function]: PhotosController->index()
#12 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#13 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#14 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#16 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#17 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#18 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#19 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#20 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#21 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#22 {main} [] []
[2013-05-15 20:00:22] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dd()' in /home/gristech/l4/app/routes.php:15
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 20:01:27] log.ERROR: exception 'Illuminate\Session\TokenMismatchException' in /home/gristech/l4/app/filters.php:92
Stack trace:
#0 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#1 /home/gristech/l4/bootstrap/compiled.php(14855): call_user_func_array(Object(Closure), Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('csrf', Object(Illuminate\Http\Request), Array)
#3 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'csrf', Object(Illuminate\Http\Request))
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'postLogin')
#5 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postLogin', Array)
#6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#7 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#8 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#9 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-05-15 20:01:40] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/Confide.php(115): Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/Confide.php(115): User->where('username', '=', 'user')
#12 /home/gristech/l4/bootstrap/compiled.php(5381): Zizaco\Confide\Confide->logAttempt(Array, true)
#13 /home/gristech/l4/app/controllers/user/UserController.php(93): Illuminate\Support\Facades\Facade::__callStatic('logAttempt', Array)
#14 /home/gristech/l4/app/controllers/user/UserController.php(93): Zizaco\Confide\ConfideFacade::logAttempt(Array, true)
#15 [internal function]: UserController->postLogin()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postLogin', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postLogin', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:02:02] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11487): Illuminate\Database\DatabaseManager->connection()
#6 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/ConfideUser.php(296): Illuminate\Database\DatabaseManager->__call('table', Array)
#7 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/ConfideUser.php(296): Illuminate\Database\DatabaseManager->table('users')
#8 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/ConfideUser.php(174): Zizaco\Confide\ConfideUser->generateUuid('users', 'confirmation_co...')
#9 /home/gristech/l4/vendor/laravelbook/ardent/src/LaravelBook/Ardent/Ardent.php(203): Zizaco\Confide\ConfideUser->beforeSave(false)
#10 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/ConfideUser.php(243): LaravelBook\Ardent\Ardent->save(Array, Array, Array, NULL, NULL)
#11 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/ConfideUser.php(159): Zizaco\Confide\ConfideUser->real_save(Array, Array, Array, NULL, NULL)
#12 /home/gristech/l4/app/controllers/user/UserController.php(38): Zizaco\Confide\ConfideUser->save()
#13 [internal function]: UserController->postIndex()
#14 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#15 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postIndex', Array)
#16 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postIndex', Array)
#17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#18 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#19 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#20 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#21 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#22 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#24 {main} [] []
[2013-05-15 20:02:21] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'blogs')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'blogs')
#15 [internal function]: BlogController->getView('blogs')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blogs')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:02:29] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'users')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'users')
#15 [internal function]: BlogController->getView('users')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('users')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:03:03] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/Confide.php(115): Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/Confide.php(115): User->where('username', '=', 'user')
#12 /home/gristech/l4/bootstrap/compiled.php(5381): Zizaco\Confide\Confide->logAttempt(Array, true)
#13 /home/gristech/l4/app/controllers/user/UserController.php(93): Illuminate\Support\Facades\Facade::__callStatic('logAttempt', Array)
#14 /home/gristech/l4/app/controllers/user/UserController.php(93): Zizaco\Confide\ConfideFacade::logAttempt(Array, true)
#15 [internal function]: UserController->postLogin()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postLogin', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postLogin', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:03:08] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/Confide.php(115): Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 /home/gristech/l4/vendor/zizaco/confide/src/Zizaco/Confide/Confide.php(115): User->where('username', '=', 'user')
#12 /home/gristech/l4/bootstrap/compiled.php(5381): Zizaco\Confide\Confide->logAttempt(Array, true)
#13 /home/gristech/l4/app/controllers/user/UserController.php(93): Illuminate\Support\Facades\Facade::__callStatic('logAttempt', Array)
#14 /home/gristech/l4/app/controllers/user/UserController.php(93): Zizaco\Confide\ConfideFacade::logAttempt(Array, true)
#15 [internal function]: UserController->postLogin()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postLogin', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postLogin', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:03:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('public-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('public-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 20:04:36] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('orderBy', Array)
#11 [internal function]: Post->orderBy('created_at', 'DESC')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(13): Illuminate\Database\Eloquent\Model::__callStatic('orderBy', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(13): Post::orderBy('created_at', 'DESC')
#15 [internal function]: BlogController->getIndex()
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:05:08] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(11278): Illuminate\Database\Eloquent\Model->newQuery()
#10 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#11 [internal function]: Post->where('slug', '=', 'surl')
#12 /home/gristech/l4/bootstrap/compiled.php(11291): call_user_func_array(Array, Array)
#13 /home/gristech/l4/app/controllers/BlogController.php(29): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#14 /home/gristech/l4/app/controllers/BlogController.php(29): Post::where('slug', '=', 'surl')
#15 [internal function]: BlogController->getView('surl')
#16 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#17 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#18 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#19 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('surl')
#20 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#21 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#22 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#23 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#24 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#25 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#26 {main} [] []
[2013-05-15 20:11:35] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#11 [internal function]: PhotosController->index()
#12 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#13 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#14 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#16 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#17 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#18 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#19 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#20 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#21 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#22 {main} [] []
[2013-05-15 21:55:56] log.ERROR: exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'gristech_foo'@'localhost' (using password: YES)' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:47
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(47): PDO->__construct('mysql:host=loca...', 'gristech_foo', 'bar', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 /home/gristech/l4/bootstrap/compiled.php(11553): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 /home/gristech/l4/bootstrap/compiled.php(11409): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 /home/gristech/l4/bootstrap/compiled.php(11381): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 /home/gristech/l4/bootstrap/compiled.php(11121): Illuminate\Database\DatabaseManager->connection(NULL)
#6 /home/gristech/l4/bootstrap/compiled.php(11092): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 /home/gristech/l4/bootstrap/compiled.php(10435): Illuminate\Database\Eloquent\Model->getConnection()
#8 /home/gristech/l4/bootstrap/compiled.php(10389): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Model->newQuery()
#10 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#11 [internal function]: PhotosController->index()
#12 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#13 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#14 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#16 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#17 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#18 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#19 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#20 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#21 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#22 {main} [] []
[2013-05-15 21:57:26] log.ERROR: exception 'Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'gristech_l4.photos' doesn't exist (SQL: select * from `photos`) (Bindings: array (
))' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php:494
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php(467): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select * from `...', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php(261): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(913): Illuminate\Database\Connection->select('select * from `...', Array)
#3 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(317): Illuminate\Database\Query\Builder->get(Array)
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(98): Illuminate\Database\Eloquent\Builder->getModels(Array)
#5 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Builder->get(Array)
#6 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#7 [internal function]: PhotosController->index()
#8 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#9 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#10 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#12 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#13 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#14 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2013-05-15 22:12:46] log.ERROR: exception 'Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'gristech_l4.photos' doesn't exist (SQL: select * from `photos`) (Bindings: array (
))' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php:494
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php(467): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select * from `...', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php(261): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(913): Illuminate\Database\Connection->select('select * from `...', Array)
#3 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(317): Illuminate\Database\Query\Builder->get(Array)
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(98): Illuminate\Database\Eloquent\Builder->getModels(Array)
#5 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Builder->get(Array)
#6 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#7 [internal function]: PhotosController->index()
#8 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#9 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#10 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#12 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#13 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#14 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2013-05-15 22:12:49] log.ERROR: exception 'Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'gristech_l4.photos' doesn't exist (SQL: select * from `photos`) (Bindings: array (
))' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php:494
Stack trace:
#0 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php(467): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select * from `...', Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Connection.php(261): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(913): Illuminate\Database\Connection->select('select * from `...', Array)
#3 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(317): Illuminate\Database\Query\Builder->get(Array)
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(98): Illuminate\Database\Eloquent\Builder->getModels(Array)
#5 /home/gristech/l4/bootstrap/compiled.php(9679): Illuminate\Database\Eloquent\Builder->get(Array)
#6 /home/gristech/l4/app/controllers/PhotosController.php(14): Illuminate\Database\Eloquent\Model::all()
#7 [internal function]: PhotosController->index()
#8 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#9 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#10 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#12 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#13 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#14 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2013-05-15 22:54:22] log.ERROR: exception 'ErrorException' with message 'Undefined index: argv' in /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php:57
Stack trace:
#0 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php(57): Illuminate\Exception\Handler->handleError(8, 'Undefined index...', '/home/gristech/...', 57, Array)
#1 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Application.php(111): Symfony\Component\Console\Input\ArgvInput->__construct()
#2 /home/gristech/l4/artisan(58): Symfony\Component\Console\Application->run()
#3 {main} [] []
[2013-05-15 22:55:36] log.ERROR: exception 'ErrorException' with message 'Undefined index: argv' in /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php:57
Stack trace:
#0 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php(57): Illuminate\Exception\Handler->handleError(8, 'Undefined index...', '/home/gristech/...', 57, Array)
#1 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Application.php(111): Symfony\Component\Console\Input\ArgvInput->__construct()
#2 /home/gristech/l4/artisan(58): Symfony\Component\Console\Application->run()
#3 {main} [] []
[2013-05-15 22:57:57] log.ERROR: exception 'ErrorException' with message 'Undefined index: argv' in /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php:57
Stack trace:
#0 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php(57): Illuminate\Exception\Handler->handleError(8, 'Undefined index...', '/home/gristech/...', 57, Array)
#1 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Application.php(111): Symfony\Component\Console\Input\ArgvInput->__construct()
#2 /home/gristech/l4/artisan(58): Symfony\Component\Console\Application->run()
#3 {main} [] []
[2013-05-15 22:58:37] log.ERROR: exception 'ErrorException' with message 'Undefined index: argv' in /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php:57
Stack trace:
#0 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php(57): Illuminate\Exception\Handler->handleError(8, 'Undefined index...', '/home/gristech/...', 57, Array)
#1 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Application.php(111): Symfony\Component\Console\Input\ArgvInput->__construct()
#2 /home/gristech/l4/artisan(58): Symfony\Component\Console\Application->run()
#3 {main} [] []
[2013-05-15 23:02:00] log.ERROR: exception 'ErrorException' with message 'Undefined index: argv' in /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php:57
Stack trace:
#0 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Input/ArgvInput.php(57): Illuminate\Exception\Handler->handleError(8, 'Undefined index...', '/home/gristech/...', 57, Array)
#1 /home/gristech/l4/vendor/symfony/console/Symfony/Component/Console/Application.php(111): Symfony\Component\Console\Input\ArgvInput->__construct()
#2 /home/gristech/l4/artisan(58): Symfony\Component\Console\Application->run()
#3 {main} [] []
[2013-05-15 23:20:19] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 23:20:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 23:20:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 23:20:56] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Andrew13\Helpers\Tidy' not found' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:25
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:21:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Andrew13\Helpers\Tidy' not found' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:25
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:21:51] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Andrew13\Helpers\Tidy' not found' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:25
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:22:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:1120
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(5379): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/l4/app/controllers/BlogController.php(38): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/l4/app/controllers/BlogController.php(38): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('blogs')
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blogs')
#8 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-05-15 23:22:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:1120
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(5379): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/l4/app/controllers/BlogController.php(38): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/l4/app/controllers/BlogController.php(38): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('blog')
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blog')
#8 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-05-15 23:23:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:1120
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(5379): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/l4/app/controllers/BlogController.php(38): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/l4/app/controllers/BlogController.php(38): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('blog')
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blog')
#8 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-05-15 23:23:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:1120
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(5379): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/l4/app/controllers/BlogController.php(39): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/l4/app/controllers/BlogController.php(39): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('blog')
#4 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('blog')
#8 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-05-15 23:24:11] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings-white.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings-white.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:28] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dd()' in /home/gristech/l4/app/controllers/BlogController.php:12
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:24:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Array)
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings-white.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:24:53] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dd()' in /home/gristech/l4/app/controllers/BlogController.php:12
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:25:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings-white.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:25:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:25:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Andrew13\Helpers\Tidy' not found' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:25
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:27:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings-white.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:27:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Asset [assets/img/glyphicons-halflings.png] was unable to be processed.' in /home/gristech/l4/vendor/jasonlewis/basset/src/Basset/Output/Controller.php:45
Stack trace:
#0 [internal function]: Basset\Output\Controller->processAsset('admin-css', 'assets/img/glyp...')
#1 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#2 /home/gristech/l4/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('processAsset', Array)
#3 /home/gristech/l4/bootstrap/compiled.php(8312): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'processAsset', Array)
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('admin-css', 'assets/img/glyp...')
#5 /home/gristech/l4/bootstrap/compiled.php(14789): call_user_func_array(Object(Closure), Array)
#6 /home/gristech/l4/bootstrap/compiled.php(14768): Illuminate\Routing\Route->callCallable()
#7 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2013-05-15 23:27:31] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:1120
Stack trace:
#0 /home/gristech/l4/vendor/zizaco/entrust/src/Zizaco/Entrust/Entrust.php(118): Illuminate\Foundation\Application->abort(404)
#1 [internal function]: Zizaco\Entrust\Entrust->Zizaco\Entrust\{closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(14855): call_user_func_array(Object(Closure), Array)
#3 /home/gristech/l4/bootstrap/compiled.php(14805): Illuminate\Routing\Route->callFilter('admin_b1b79d', Object(Illuminate\Http\Request))
#4 /home/gristech/l4/bootstrap/compiled.php(14766): Illuminate\Routing\Route->callBeforeFilters(Object(Illuminate\Http\Request))
#5 /home/gristech/l4/bootstrap/compiled.php(8332): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#6 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#8 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#9 {main} [] []
[2013-05-15 23:27:49] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Andrew13\Helpers\Tidy' not found' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:25
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:28:09] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Andrew13\Helpers\tidy' not found' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:25
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:29:17] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in /home/gristech/l4/app/library/Andrew13/Helpers/String.php:52
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-05-15 23:45:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-05-15 23:45:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/l4/bootstrap/compiled.php:8654
Stack trace:
#0 /home/gristech/l4/bootstrap/compiled.php(8355): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/l4/bootstrap/compiled.php(8331): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/l4/bootstrap/compiled.php(989): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/l4/bootstrap/compiled.php(972): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/l4/public/index.php(51): Illuminate\Foundation\Application->run()
#5 {main} [] []
